### DISCUSSION TITLE
Steam notifications issues

### DISCUSSION THREAD

**Discussion by pxy1337** [i3/i3#6113] [Q&A]
Hi,
every time I get a steam notification(like someone starts a game or I get a dm) notification pops up in bottom right, but then when it disappears my windows flicker for a while because of some resizing I think.

How could I fix it?
I tried to find a solution to it, but couldn't find any here.

Thank u in advance and lmk if u need more info :)

**Comment by gratux** [comment]
I believe i have the same issue. Just after the notification disappears, a new window is opened and immediately closed. other windows get resized twice because of this.

**Comment by luinbytes** [comment]
Having this same problem, might be fixable with a window rule? All my steam windows are forced to float by default already though.

**Comment by Ahrotahntee** [comment]
The problem is whatever the toast is doing as it's self-dismissing, the window that flashes up has literally no attributes set. It has no name, no class, no hints, etc.

I have a line in i3 config `for_window [class="^$" title="^$"] floating enable` which is preventing it from resizing my focused window or taking focus. I'm just *waiting* for it to cause a problem for me elsewhere though, so if you're going to take my bad advice keep an eye out for weird floating windows later on.

**Comment by Anthony1x** [comment]
Chiming in to say that this is not an i3 specific issue, I'm on qtile and am experiencing the same problem. Maybe this issue should be directed at valve instead?

**Comment by DrWarpMan** [comment]
I am not sure what changed, but I am no longer experiencing this bug. Steam must have fixed it.
Can anyone else confirm?

**Reply by luinbytes** [reply]
Can confirm, booted a i3 Arch instance and it seems to pop up normally now.

Valve are now working with arch btw, so maybe we expect big fixes from now on B)
